.travis.yml

```diff
@@ -10,7 +10,6 @@ php:
 
 before_install:
   - mv ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini{,.disabled} || echo "xdebug not available"
-  - travis_retry composer self-update
```

Comment:
Could this possibly be related to the failure?

Reply:
Shouldn't. It's updated automatically. I'll skip this for tonight and test it tomorrow.